packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,7 +146,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root' className={ css('ms-Dropdown-container') }>
         { label && (
-          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
+          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } htmlFor={ id } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
@mdahamiwal, `aria-label` doesn't indicate what the `<label>` element is labeling, so I would argue that this is still needed even if `aria-label` exists. 

I think we should only remove `aria-label` when the `label` prop exists, but always have `htmlFor` set on the `<label>` element prop if it is shown.